src/main/resources/application.properties

```diff
@@ -143,6 +143,12 @@ cache.appsettings.configuration.eternal=false
 cache.appsettings.configuration.timeToIdleSeconds=0
 cache.appsettings.configuration.timeToLiveSeconds=60
 
+
+cache.rocketchat.configuration.maxEntriesLocalHeap=100
+cache.rocketchat.configuration.eternal=false
+cache.rocketchat.configuration.timeToIdleSeconds=0
+cache.rocketchat.configuration.timeToLiveSeconds=900
```

Comment:
hey @tkuzynow, I have learnt that in Caritas the displayname cannot be changed. I have understood that this request is only made because of that. Would it then make sense to cache this data for much longer?

//CC @Leandro13Silva13